File: 프로그래머스/0/181867. x 사이의 개수/x 사이의 개수.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(string myString) {
    vector<int> answer;
    int count=0;
    
    for(char c:myString)
    {
        if(c!='x')
            count++;
        else
        {
            answer.push_back(count);
            count=0;
        }
    }
    
    answer.push_back(count);
    
    return answer;
}

File: 프로그래머스/0/181867. x 사이의 개수/x 사이의 개수.py
def solution(myString):
    answer,tmp = [],''
    
    for char in myString:
        if char!='x':
            tmp+=char
        else:
            if len(tmp)!=0:
                answer.append(len(tmp))
                tmp=''
            else:
                answer.append(0)
    
    if len(tmp)!=0:
        answer.append(len(tmp))
    else:
        answer.append(0)
    return answer